test/ServerFactoryTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Swoole;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Swoole\Process;
+use Zend\Expressive\Swoole\ServerFactory;
+
+class ServerFactoryTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->container = $this->prophesize(ContainerInterface::class);
+        $this->serverFactory = new ServerFactory();
+    }
+
+    public function testInvokeWithoutConfig()
```

Comment:
Note to self: this is "testFactoryCreatesInstanceUsingDefaultsWhenNoConfigIsAvailable".